Core/clim-core/commands/commands.lisp

```diff
@@ -406,6 +407,25 @@ designator) inherits menu items."
                          command-name
                          command-line-name
                        &allow-other-keys)
+  (ecase type
+    (:command
+     ;; This is specified to be a cons, but McCLIM is more permissive
+     ;; (and we don't want to break the backward compatibility).
+     (unless (consp value)
+       (setf value (list value))))
+    (:function
+     ;; A function of two arguments (funcalled).
+     (check-type value (or function symbol (cons (eql setf) null))))
```

Comment:
`(cons (eql setf) null)` looks wrong, wouldn't that be just `(setf)`. I think it should be `(cons (eql setf) (cons symbol null))`.

Reply:
fixed by adding a type to utils